addons/mail/static/src/messaging/component/messaging_menu/messaging_menu_tests.js

```diff
@@ -226,6 +227,38 @@ QUnit.test('basic rendering', async function (assert) {
     );
 });
 
+QUnit.test('counter', async function (assert) {
```

Comment:
name of test is too generic. Here it tests that a single isolated failure notification is included in messaging menu (global) counter.

Reply:
counter is taking into account failure notification